### ISSUE TITLE
new account - username "rule"

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#1589]
Provide instructions for creating operator-compliant username/password for all accounts; make it look like a rule but hard enforcement isn't necessary (@ccicero @Jegelewicz )

**Comment by Jegelewicz** [comment]
I'm reviving this because of https://github.com/ArctosDB/data-migration/issues/72

Can we please just make the rules the same for everyone? This has been an issue with every incoming collection.

**Comment by campmlc** [comment]
I agree with this. Supporting our new users is more important than making
downloads one click easier for the public.

**Comment by ccicero** [comment]
I'm not following. What's the issue where they are not the same for everyone? 

Whenever I work with someone to create an account, I tell them that the username can't have a period (Arctos doesn't like that!) and that the pwd should have at least 8 characters, a combo of numbers and letters, and at least one symbol like "!"

If they follow that 'rule' then they usually authenticate without a problem. If they don't, then there are often authentication issues and they need to change their pwd.

**Comment by dustymc** [comment]
I don't think this is realistic - some significant percentage of our ~4K users don't comply with operator requirements, and don't have any reason to.

The requirements are all documented, and there's a big READ THIS! link beside the invite link. This is a potentially dangerous level of access, and anyone with this access needs to understand the corresponding documentation.

Perhaps this is a good place for a short how-to video?

I can check usernames before invitation and add more links to the documentation or something for users who won't validate.

The password rules are documented in the form that accepts invitations, and there's a real-time check. There's no way for the inviter to know if the invitee's password will work before hand, but that won't stop the invitation.

**Comment by ccicero** [comment]
A short video would be great. It will save a lot of headache for operators who are trying to authenticate!

**Comment by Jegelewicz** [comment]
> I'm not following. What's the issue where they are not the same for everyone?

They are not and many of our new collection managers already have Arctos login IDs that do not comply with operator account requirements.

**Comment by dustymc** [comment]
The admin form is spiffied up a bit and has a new "i've read the docs" widget. Leaving this open in the hopes of a tutorial.

EDIT: rebuilt triggers as well
